index.html

```diff
@@ -0,0 +1,268 @@
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+          <!-- Project Card five -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+          <!-- Project Card Six -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- About Section -->
+      <section id="about-section">
+        <h4 id="about-intro">About me</h4>
+        <p id="about-desc">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+        <div><button>Get My Resume</button></div>
+        <!-- Tools Section -->
+        <div class="tools">
+          <!-- for languages -->
+          <div class="tool-card">
+            <img src="./Image/Icons.svg" alt="" />
+            <h4 class="tool-name">Languages</h4>
+            <ul class="tool-list">
+              <li>JavaScript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>CSS</li>
+            </ul>
+          </div>
+          <!-- for Frameworks -->
+          <div class="tool-card">
+            <img src="./Image/icon-frameworks.svg" alt="" />
+            <h4 class="tool-name">Frameworks</h4>
+            <ul class="tool-list">
+              <li>JavaScript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>CSS</li>
+            </ul>
+          </div>
+          <!-- for skills -->
+          <div class="tool-card">
+            <img src="./Image/Icons-skills.svg" alt="" />
+            <h4 class="tool-name">Languages</h4>
+            <ul class="tool-list">
+              <li>JavaScript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>CSS</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <!-- contact setion -->
```

Comment:
Let's wrap all the technology sections in the `<ul>` tag and all the sections like languages, frameworks and skills will be as `<li>` since they are a list. What do you think about this?